packages/file-explorer/src/FileTestUtils.ts

```diff
@@ -0,0 +1,98 @@
+import { DirectoryStorageItem, FileStorageItem } from './FileStorage';
+
+class FileTestUtils {
```

Comment:
I've been moving to a different format for "utils" files, see `ClipboardUtils` as an example.
Instead of declaring all the methods as statics on a class, just export each individual function, then export them all as the default, e.g.
```
export function makeFileName(...) { ... }

export function makeDirName( ... ) { ... }

export default { makeFileName, makeDirName };
```

There's a few reasons why I like it better:
- Consumers can just import the methods they need, and it shortens up their code. Instead of:
```
import FileTestUtils from './FileTestUtils';

const file = FileTestUtils.makeFileName(...);
```
It's now just:
```
import { makeFileName } from './FileTestUtils';

const fileName = makeFileName(...);
```
- Doesn't allow the methods to be re-assigned. Technically, someone could write `FileTestUtils.makeFileName = function() { ... }` which could screw stuff up for every other consumer.
- Clear how the utils are supposed to be used. Right now you could do `const fileTestUtils = new FileTestUtils()` which doesn't make sense; you can't really do anything with the class, you just want the bag of functions.